visualize_grasp:
    grasp:
        grasp_frame: ptu_camera_color_optical_frame
    camera:
        info_topic: /ptu_camera/camera/color/camera_info
        color_topic: /ptu_camera/camera/color/image_raw
        depth_topic: /ptu_camera/camera/aligned_depth_to_color/image_raw
        camera_frame: ptu_camera_color_optical_frame
    input:
        grasp_topic: /grasp_input_topic
        grasps_topic: /grasps_input_topic
    output:
        image_points: /grasp_lib/drawn_grasp
        marker_topic: /grasp_lib/markers
        method_names: /grasp_lib/names
    options:
        draw_image: True
